google-cast/src/main/java/com/theoplayer/sample/playback/cast/PlayerActivity.java

```diff
@@ -41,14 +43,13 @@ protected void onCreate(Bundle savedInstanceState) {
         // Gathering THEO objects references.
         theoPlayer = viewBinding.theoPlayerView.getPlayer();
 
-        // Add Cast integration.
+        // THEOplayer automatically adds all available integrations to the player via the autoIntegrations() configuration. Here, we add the cast integration manually to configure the cast strategy.

```

Comment:
Pro-tip: you can now also use `THEOplayerConfig.Builder.castConfiguration()` to set the Chromecast configuration, without having to manually add the cast integration. 😉 

That said, I don't know whether that would make things easier in this particular case. 😅